src/Host.cpp

```diff
@@ -1216,7 +1220,7 @@ void Host::readPackageConfig(const QString& luaConfig, QString& packageName)
             qDebug() << reason.c_str() << " in config.lua:" << e.c_str();
```

Comment:
Ah, you will need a space after the `:` in `" in config.lua:"` otherwise the `e.c_str()` will not have a space between it and that.